### DISCUSSION TITLE
Cover image support?

### DISCUSSION THREAD

**Discussion by howesteve** [importantimport/urara#57] [General]
Hello,

Nice project. I was wondering if there is cover image support? Like similar projects do, i.e. you define an image that will show up on post listing and/or when you open the post.

**Comment by Interstellar750** [comment]
Is this the cover image you are referring to?

In the [demo page](https://urara-demo.netlify.app/) it looks like this:
![IMG_20230215_102303.jpg](https://user-images.githubusercontent.com/38517305/218912193-bd17ef58-b8a5-4767-bf7f-07ceb99ef600.jpg)

It can be defined in the article metadata or called frontmatter, see: [urara/hello-world/+page.svelte.md#L3](https://github.com/importantimport/urara/blob/main/urara/hello-world/+page.svelte.md?plain=1#L3)
